### ISSUE TITLE
skate link freezes

### ISSUE THREAD

**Issue by 174n** [charmbracelet/skate#60]
After I type skate link on one device and the code on another device I get "Link request from:" dialog with yes and no buttons. I can switch between them by using arrow keys. Enter to select didn't work, but after I pressed Y the bash got frozen, arrow keys no longer work and even Ctrl+C doesn't have any effect.

On one device I have raspbian and on another arch.

**Comment by bashbunni** [comment]
What version of skate are you using? Are you still encountering this issue?

**Comment by bashbunni** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Skate Link Freezes](https://trello.com/c/XF7sPw3Q/8-skate-link-freezes)

**Comment by bashbunni** [comment]
Hey @174n is this still a problem for you?

**Comment by tonietto** [comment]
Hey @bashbunni, it is for me. I am using skate version `v0.2.1`, fresh installed on both machines (1 macOS and 1 Arch Linux).

**Comment by bashbunni** [comment]
We had another user in our Discord mentioning some issues with Arch and Skate. I believe there might be more to troubleshoot here with Arch and Skate... I wasn't able to reproduce this issue linking between Pop_OS! and macOS. I'll see about getting an Arch VM going to try and reproduce.

**Comment by TravisSpangle** [comment]
I'm experiencing this and I think the issue is iTerm.

||computer 1|computer 2|
|--|--|---|
|Mac OS|14.1.1|13.2.1|
|iTerm2|3.4.23|3.4.23|
|Skate|v0.2.1 (ef9b184)|v0.2.1 (ef9b184)|

I experienced this and believe I isolated it to iterm2.  I experienced the hanging behavior in every situation except:

comp 1 used terminal (not iterm2) as the initiation.
comp 2 used iterm2 to link.

When I select and enter on Yes the prompt clears and messages me that I have been linked.

So I would experience the stuck prompt on any laptop using iTerm2 as the initiation.

**Comment by bashbunni** [comment]
@TravisSpangle are you linking a macOS machine to an arch machine?

**Comment by TravisSpangle** [comment]
both are macOS machines. Have I stumbled into the wrong tooling?

**Comment by bashbunni** [comment]
@TravisSpangle No, I just thought the issue was possibly arch-related since I wasn't able to reproduce. Thanks! I can try this on my end as well and see if I can reproduce. Thank you so much for the additional info

**Comment by caarlos0** [comment]
Hey!

As of 1.0.0, we removed the cloud features, so this is no longer an issue.

Closing!

https://github.com/charmbracelet/skate/releases/tag/v1.0.0
